packages/react/src/components/DashboardEditor/DashboardEditorCardRenderer.test.jsx

```diff
@@ -230,4 +230,71 @@ describe('DashboardEditorCardRenderer', () => {
     );
     expect(screen.getByText(/defaultCard/)).toBeInTheDocument();
   });
+
+  it('should call getValidDataItems when passed', () => {
+    const getValidDataItems = jest.fn();
+    render(
+      <DashboardEditorCardRenderer
+        {...commonProps}
+        size={CARD_SIZES.MEDIUM}
+        getValidDataItems={getValidDataItems}
+      />
+    );
+
+    expect(getValidDataItems).toHaveBeenCalledWith({ size: CARD_SIZES.MEDIUM });
+  });
+
+  // it('should call onCardChange when image card is updated', () => {
```

Comment:
Remove this one?

Reply:
yeah, not sure how that commit related to DashboardEditorCardRenderer got included...I may have accidentally created this branch from that one or something. I've revert that commit in this PR.